recipes/h3/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "h3" %}
+{% set version = "3.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/uber/h3/archive/v{{ version }}.tar.gz
+  sha256: 92fad4467e0af343dbf3d003da09bd2df6a1350f46fcc569ca33648593d78f25
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - conda-verify
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libh3${SHLIB_EXT}
```

Comment:
Can you also check that `$PREFIX/include/h3/h3api.h` exists?

Reply:
The pipeline is green

Reply:
Can you add this check?

Reply:
as this is already merged (and this check has been added by you, I will resolve it)